Sources/ProjectSpec/Scheme.swift

```diff
@@ -239,16 +244,60 @@ public struct Scheme: Equatable {
     }
 
     public struct BuildTarget: Equatable, Hashable {
-        public var target: String
+        public var target: TargetReference
         public var buildTypes: [BuildType]
 
-        public init(target: String, buildTypes: [BuildType] = BuildType.all) {
+        public init(target: TargetReference, buildTypes: [BuildType] = BuildType.all) {
             self.target = target
             self.buildTypes = buildTypes
         }
     }
 }
 
+
+public struct TargetReference: Equatable, Hashable {
+    public let name: String
+    public let location: Location
+
+    public enum Location: Equatable, Hashable {
+        case local
+        case project(String)
+    }
+
+    public init(name: String, location: Location = .local) {
+        self.name = name
+        self.location = location
+    }
+}
+
+extension TargetReference {
+    public init(string: String) throws {
+        let paths = string.split(separator: "/")
+        guard paths.count <= 2 && !paths.isEmpty else {
+            throw SpecParsingError.invalidTargetReference(string)
+        }
+        switch paths.count {
+        case 2:
+            location = .project(String(paths[0]))
+            name = String(paths[1])
+        case 1:
+            location = .local
+            name = String(paths[0])
+        default: fatalError("unreachable")
+        }
+    }
+}
+
+extension TargetReference {
+    public func toString() -> String {
```

Comment:
Let's make this a `reference: String` property, and also conform to CustomStringConvertable that uses this in it's description